### DISCUSSION TITLE
How to set logo size in reveal.js?

### DISCUSSION THREAD

**Discussion by lizhiwei1994** [quarto-dev/quarto-cli#2209] [Q&A]
Hi everyone.

I know use `logo: "https://www.rstudio.com/wp-content/uploads/2018/10/RStudio-Logo-Flat.png"` can set  a logo in each slides.

But how can I change the logo size. My logo is too samll that can not be seen clearly in my slides.

Below is my logo png file.

![logo_ccmu](https://user-images.githubusercontent.com/59855957/187441375-ac0b7654-6147-4999-8651-1af9abb618c1.png)

**Comment by cderv** [comment]
`.reveal .slide-logo` would be the CSS to change. We are setting `max-height: 2.2rem` so that the logo stays in the footer size and not take slide space content.

But if you want to increase just change this rule. For example 

````css
.reveal .slide-logo {
   max-height: 4em !important;
}
```` 

`!important` is key here also because the CSS is defined in `footer.css` in the first place and this come after a user provide style customization

![image](https://user-images.githubusercontent.com/6791940/187445389-16eba60d-1540-4620-aa01-16ae20fad8fe.png)

<details><summary> Code </summary>

````markdown
---
title: "Untitled"
format: 
  revealjs:
    logo: "https://user-images.githubusercontent.com/59855957/187441375-ac0b7654-6147-4999-8651-1af9abb618c1.png"
    theme: [default, styles.scss]
---

## Title

Test

````

</details>

**Reply by cderv** [reply]
By the way, more context to this answer in here: https://github.com/quarto-dev/quarto-cli/issues/746#issuecomment-1111203492

**Reply by lizhiwei1994** [reply]
@cderv , Thanks a lot. It really helps me!
